README.md

```diff
@@ -6,10 +6,56 @@
 [![License](https://img.shields.io/cocoapods/l/SweetSwift.svg?style=flat)](https://cocoapods.org/pods/DATAStack)
 
 
-## Usage
+## Enums
+
+**SweetSwift** adds a count variable to your Enum so you can avoid having a special case for this.
+
+A common scenario would be that you use enums in your UITableViewController to separate the different sections, for example:
+
+```swift
+enum SectionType: Int {
+    case title, price, others, delete, count
+}
+
+override func numberOfSectionsInTableView(tableView: UITableView) -> Int {
+    return SectionType.count.rawValue
+}
+```
+
+After adding **SweetSwift** you would do something like this instead:
```

Comment:
I think you can better say: "you can do this instead". In reference to the prior example the code below is just what you would do instead with SweetSwift, not something like it but exactly that.